generator/generator.yml

```diff
@@ -316,6 +316,7 @@ modules:
       - prtMarkerSuppliesLevel
       - prtMarkerSuppliesMaxCapacity
       - prtMarkerSuppliesType
+      - prtMarkerSuppliesDescription
```

Comment:
Please put this above `prtMarkerSuppliesLevel` so things stay alphabetically sorted.

Reply:
Good point, done.